README.md

```diff
@@ -155,6 +155,14 @@ We welcome contributions to the project, visit our contributing
 - Drop us a line at hello@vizzuhq.com
 - Follow us on Twitter: [VizzuHQ](https://twitter.com/VizzuHQ)
 
+## Usage Statistics
+
+`ipyvizzu` collects aggregate usage statistics by default to follow the progress
+and overall trends of our library. We do not track, collect, or store any
+personal data or personally identifiable information. For more details, please
+visit
```

Comment:
We should mention here that it is optional. We should also mention that publishing anything made with ipyvizzu is GDPR compatible even when stats enabled.

Reply:
@simzer updated the description, please approve if it is mergeable